content/blog/2021/containerqueries.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Getting Started with Container Queries
+sub: Now is the time to begin experimenting with a long requested layout tool. 
+author: davidh
+date: 2021-04-02
+image:
+  src: blog/2021/widequote.jpg
+  alt: blockquote with magenta background and polygonal clip-path
+tags:
+  - CSS
+  - CSSWG
+  - Article
+  - Container Queries
+summary: |
+  The `@container` query, that elusive feature developers have been requesting and proposing for years, has finally made its debut in browsers. Well, sort of. What are **container queries** exactly and how do they work?
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+  In the latest version of [Chrome Canary](https://www.google.com/chrome/canary), the most recent `@container` query proprosal, developed by OddBird's own [Miriam Suzanne](https://www.oddbird.net/authors/miriam/), is available for use behind an experimental flag. Miriam's [proposal](https://github.com/w3c/csswg-drafts/issues/5796) builds upon the ideas previously proposed by others, in particular David Baron, who wrote the [original draft](https://github.com/dbaron/container-queries-implementability). 
```

Comment:
Accidental indent? 

(this sentence also takes a few turns, which could maybe be simplified? But not a big issue)

Reply:
Made an attempt to simplify here. I think it reads better. Fixed the indent.